src/errors.rs

```diff
@@ -123,12 +122,6 @@ fn test_unknown_crypto_from_decode_error() {
         UnknownCryptoError::from(err_two)
     );
     assert_eq!(err, "UnknownCryptoErrorUnknownCryptoError");
```

Comment:
Just out of curiosity, why is `UnknownCryptoError` repeated twice?

Reply:
In continuation of the previous comment, the two formatters are `{:?}{}`, which means `"UknownCryptoError" is "printed two times in the string", once using the `Debug` impl, the other using the `Display` impl. I'll separate these with a `":"`, perhaps that would make it clearer.

Reply:
Thanks for reviewing @vlmutolo, always a big help. I'll add the separator mentioned above. Let me know if it's not clearer or something else would be better. I'll merge it with that for now though, so I can get this included for the fuzzing.